sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/HDFSBackedStateStoreProvider.scala

```diff
@@ -126,6 +126,7 @@ private[sql] class HDFSBackedStateStoreProvider extends StateStoreProvider with
         valueSchema: StructType,
         keyStateEncoderSpec: KeyStateEncoderSpec,
         useMultipleValuesPerKey: Boolean = false,
```

Comment:
TODO: as discussed with Anish today, will made this a ENUM type and passed into RocksDB

Reply:
Will not use ENUM. Keep the Boolean, use virtual col family by default if the boolean is true